app/src/main/java/com/example/aaa/pinkcalculator/MainActivity.java

```diff
@@ -75,202 +76,189 @@ private void nextOperation() {
-                if (Numbers.canAddNumberToFirsNumberField(this)) {
-                    fieldForFirsNumber.append("7");
+                if (Numbers.canAddNumberToFirstNumberView(this)) {
+                    firstNumberView.append(SEVEN);
+                } else if (Numbers.canAddNumberToSecondNumberView(this)) {
+                    secondNumberView.append(SEVEN);
                 }
-                else if (Numbers.canAddNumberToSecondNumberField(this)) {
-                    fieldForSecondNumber.append("7");
-                }
-                break;
 
-            case R.id.b_clickingOn_8:
+                break;
+            case R.id.eight_button:
                 nextOperation();
-                if (Numbers.canAddNumberToFirsNumberField(this)) {
-                    fieldForFirsNumber.append("8");
-                }
-                else if (Numbers.canAddNumberToSecondNumberField(this)) {
-                    fieldForSecondNumber.append("8");
+                if (Numbers.canAddNumberToFirstNumberView(this)) {
+                    firstNumberView.append(EIGHT);
+                } else if (Numbers.canAddNumberToSecondNumberView(this)) {
+                    secondNumberView.append(EIGHT);
                 }
-                break;
 
-            case R.id.b_clickingOn_9:
+                break;
+            case R.id.nine_button:
                 nextOperation();
-                if (Numbers.canAddNumberToFirsNumberField(this)) {
-                    fieldForFirsNumber.append("9");
-                }
-                else if (Numbers.canAddNumberToSecondNumberField(this)) {
-                    fieldForSecondNumber.append("9");
+                if (Numbers.canAddNumberToFirstNumberView(this)) {
+                    firstNumberView.append(NINE);
+                } else if (Numbers.canAddNumberToSecondNumberView(this)) {
+                    secondNumberView.append(NINE);
                 }
-                break;
 
-            case R.id.b_clickingOn_point:
+                break;
+            case R.id.point_button:
                 nextOperation();
-                if (Numbers.canAddPointToFirsNumberField(this)) {
-                    if (fieldForFirsNumber.getText().length() == 0 ||
-                            fieldForFirsNumber.getText().toString().equals("-")) {
-                        fieldForFirsNumber.append("0.");
-                    }
-                    else fieldForFirsNumber.append(".");
-                }
-                else if (Numbers.canAddPointToSecondNumberField(this)) {
-                    if (fieldForSecondNumber.getText().length() == 0 ||
-                            fieldForSecondNumber.getText().toString().equals("-")) {
-                        fieldForSecondNumber.append("0.");
-                    }
-                    else fieldForSecondNumber.append(".");
+                if (Numbers.canAddPointToFirstNumberView(this)) {
+                    if (firstNumberView.getText().length() == 0 ||
```

Comment:
This block of code duplicated and can be extracted into some function

`firstNumberView.getText().length() == 0 ||  firstNumberView.getText().toString().equals(MINUS)`

Also please fix the same issue below